sirepo/feature_config.py

```diff
@@ -181,7 +181,6 @@ def b(msg, dev=False):
             "Link to Sirepo Slack workspace; uid will be appended",
         ),
         srw=dict(
-            app_url=("/en/xray-beamlines.html", str, "URL for SRW link"),
```

Comment:
The "app_url" defaults as /srw  in dev, but in prod it's /en/xray...

Reply:
Should I just change app_url to "/srw"?

Reply:
df44df69b0cf6074618593a4c872ea9c2213cc9a